tmk_core/common/keyboard.c

```diff
@@ -64,20 +64,41 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 
 #ifdef MATRIX_HAS_GHOST
+static uint16_t matrix_ghost_check[MATRIX_ROWS];
```

Comment:
It's pulling from keymaps[0][row][col] and checking for 0's, so I guess I should just read from that and take the performance hit. I don't think it would be much. I'll revise this a bit!